system_baseline/internal_views/v1.py

```diff
@@ -23,17 +23,29 @@ def get_version():
 @metrics.api_exceptions.count_exceptions()
 def get_baselines_by_system_id(system_id=None):
     account_number = view_helpers.get_account_number(request)
+    org_id = view_helpers.get_org_id(request)
 
     if system_id:
         validate_uuids([system_id])
-        query = SystemBaselineMappedSystem.query.filter(
-            SystemBaselineMappedSystem.account == account_number,
-            SystemBaselineMappedSystem.system_id == system_id,
-        )
+        if org_id:
+            query = SystemBaselineMappedSystem.query.filter(
+                SystemBaselineMappedSystem.org_id == org_id,
+                SystemBaselineMappedSystem.system_id == system_id,
+            )
+        else:
+            query = SystemBaselineMappedSystem.query.filter(
+                SystemBaselineMappedSystem.account == account_number,
```

Comment:
Alternative to full branching of query building is to branch out only the org_id vs account condition.

```
tenant_filter = SystemBaselineMappedSystem.org_id == org_id if org_id \
   else SystemBaselineMappedSystem.account == account_number

query = SystemBaselineMappedSystem.query.filter(
   tenant_filter,
   SystemBaselineMappedSystem.system_id == system_id,
)
```

Reply:
Thanks, this looks nicer